webapp/store/views.py

```diff
@@ -333,6 +333,12 @@ def snap_details(snap_name):
         icons = [
             m['url'] for m in details['snap']['media'] if m['type'] == "icon"]
 
+        default_track = '10' if details['name'] == 'node' else 'latest'
```

Comment:
Some code comment would be nice around that, so it's easier to discover that that's a node-only workaround.

Reply:
Added a comment